tests/forms_tests/widget_tests/test_multiwidget.py

```diff
@@ -118,6 +123,42 @@ def test_constructor_attrs(self):
             '<input id="bar_1" type="text" class="small" value="lennon" name="name_1" />'
         ))
 
+    def test_constructor_attrs_with_type(self):
+        attrs = {'type': 'number'}
+        widget = MyMultiWidget(widgets=(
+            TextInput,
+            TextInput,
+            TextInput(),
+        ), attrs=attrs)
+        self.check_html(widget, 'code', ['1', '2', '3'], html=(
+            '<input type="number" value="1" name="code_0" />'
+            '<input type="number" value="2" name="code_1" />'
+            '<input type="text" value="3"  name="code_2" />'
+        ))
+        widget = MyMultiWidget(widgets=(TextInput(attrs), TextInput(attrs)), attrs={'attrs': 'date'})
```

Comment:
What happened to `{'attrs': 'date'}`? On 1.10, the output is `'<input attrs="date" name="code_0" type="number" value="1" /><input attrs="date" name="code_1" type="text" value="2" />'`.